app/components/buttons/ToggleSwitch/ToggleSwitch.module.css

```diff
@@ -0,0 +1,47 @@
+.toggleSwitch {
+  display: inline-block;
+  vertical-align: middle;
+  margin-right: 0.5em;
+
+  margin-bottom: 10px;
+  width: 85px;
+}
+
+.toggleSwitchEnabled,
```

Comment:
i would get rid of the `toggleSwitch` prefix in all classnames this is needed in .less files but not in a css module
as classes are hashed and we are safe against classes conflicts